src/App.tsx

```diff
@@ -1,13 +1,33 @@
-import React from 'react';
+import React, {useState} from 'react';
 import classes from "./App.module.scss";
 import Header from "./components/Header";
 import StartPage from "./components/pages/StartPage";
+import {IOptions} from "./interfaces/options";
+import TextPage from "./components/pages/TextPage";
+import FinishPage from "./components/pages/FinishPage";
 
 const App = () => {
+    const [options, setOptions] = useState<IOptions>({
+        numberOfWords: 5,
+        startingDistance: 20,
+        lettersInWords: 7,
+        increasingDistance: 20,
+        speed: 1,
+    })
+    const [currentPage, setCurrentPage] = useState(1)
```

Comment:
Add enum, like this:
enum Step { Settings, Game, End }

And then set it instead of just hardcode numbers
const [step, setStep] = useState<Step>(Step.Settings)